lcm/src/error.rs

```diff
@@ -110,13 +110,13 @@ impl From<mpsc::RecvError> for HandleError {
 pub enum EncodeError {
     #[fail(display = "The size of array does not match the size specified in {}. Expected {}, found {}.", size_var, expected, found)]
     SizeMismatch {
-        /// The field specifying the size of the array.
+    /// The field specifying the size of the array.
```

Comment:
It did a poor job on these. I wonder why.

Reply:
I noticed that. Rustfmt has a few quirks. Overall though, it saves more trouble than it causes.